keyboards/exclusive/e6v2/bmc/keymaps/default/keymap.c

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2019 MechMerlin
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QMKBEST = SAFE_RANGE,
+  QMKURL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[0] = LAYOUT_60_ansi(
+        KC_GESC, KC_1,  KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,    KC_9,   KC_0,    KC_MINS, KC_EQL, KC_BSPC,
+        EEP_RST,  KC_Q,  KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,    KC_O,   KC_P,    KC_LBRC, KC_RBRC,KC_BSLS,
```

Comment:
Are `EEP_RST`, `RGB_TOG` and `RGB_MOD` supposed to be on the base layer?

https://github.com/qmk/qmk_firmware/blob/0152bbcc68f7d95a2aebffe0f7f9495f5ac9318c/keyboards/exclusive/e6v2/bmc/keymaps/default/keymap.c#L27-L29

Reply:
This is actually intentional. I realized that people will most likely want to test the pcb out before they solder on switches and it's incredibly difficult to reset the board or hold Fn keys without actual switches. What are your thoughts.

Reply:
use TG for a function key, instead?

Reply:
I feel very mixed about this.

You're basically requiring the user to write, compile and flash their own keymap after testing is done, though you could certainly make the argument that someone who buys this PCB would know how to do this (an E6-V2 is not a starter board for someone new to mechanical keyboards, IMO). I'm not knowledgeable enough about the community to say with confidence whether that's true or false.

My personal opinion is a default keymap should be usable for a daily driver right from the start, and I don't think this is, due to the lack of Left Shift and Tab. Perhaps a more conventional keymap should be `default`, and include this one as `(default_)testing` or something similar?

Reply:
🤔

Reply:
Maybe have a "dev" keymap for this?

Reply:
I just put the eep_rst in layer 1